### ISSUE TITLE
[FR]: Ability to affect all image URLs in articles with Article Filters

### ISSUE THREAD

**Issue by RetroAbstract** [martinrotter/rssguard#1024]
### Brief description of the feature request

This feature would allow the user to affect the URLs of **all images** in articles through Article Filters. I emphasize the "all images" as it appears some image URLs can be influenced with article filters and others can't.

I have noticed that it is image URLs that are referenced at the very top of an article in the internal viewer with "[image/jpg]" (that displays the image URL it's referencing at the bottom left on the Status bar when hovering over it with the cursor) that article filters aiming to change an article's contents don't seem to apply to.

![2](https://github.com/martinrotter/rssguard/assets/103272301/42caff9f-8743-47e0-ada6-2e9e172a7206)

Using [RSS Guard's releases feed](https://github.com/martinrotter/rssguard/releases.atom) as an example of article filters not affecting the image URLs in the article's content when applying the following filter:

```
function filterMessage() {
  msg.contents = msg.contents.replace("60", "800");
  return MessageObject.Accept;
}
```

![1](https://github.com/martinrotter/rssguard/assets/103272301/67d1b299-cf92-4cf4-bd13-e7b100f58511)

As you can see at the bottom of the article in the internal viewer, the "60" in the image URL was not chaned by the filter, even when processing the checked feed and applying the filter to the feed prior to feed-fetching. 

This is the same article used as an example in the first picture showing the "[image/jpg]" reference at the top of articles.

**Image URLs that are not referenced at the top of articles with "[image/jpg]" at the top of articles change with article filters aiming to change an article's content no problem (in my experience)**

Thanks

**Comment by martinrotter** [comment]
Yes, those images which are not affected by your fitler are actually NOT part of "contents" as they are "enclosures" (attachments) attached to the article. At this point filters do not allow to tweak existing enclosures although they allow you to add enclosures.

Theoretically, this feature could be added.

**Comment by martinrotter** [comment]
added ability to add/remove and list enclosures

https://rssguard.readthedocs.io/en/latest/features/filters.html#functions
